src/components/InputBox/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { SetStateAction, useState } from 'react';
+import EyeIcon from '../../../public/assets/icons/eye';
+import EyeCloseIcon from '../../../public/assets/icons/eyeClose';
+
+type Props = {
+    type: string;
+    name: string;
+    onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    placeholder?: string;
+    required: boolean;
+    className?: string;
+};
+
+const InputBox = (props: Props) => {
+    const [showPassword, setshowPassword] = useState<SetStateAction<boolean>>(false);
+    const { type, name, onChange, placeholder, required, className } = props;
+
+    return (
+        <>
+            <label htmlFor={name} className="block mb-2 text-sm font-medium text-gray-900 dark:text-white">
+                {name}
+            </label>
+            <div className="relative container mx-auto">
+                <input
+                    type={type == 'password' ? (showPassword ? 'text' : 'password') : 'text'}
+                    name={name}
+                    id={name}
+                    onChange={onChange}
+                    className={className || ''}
+                    placeholder={placeholder || ''}
+                    required={required}
+                />
+                {name == 'Password' && (
```

Comment:
- Should this check not be based on the input type instead of name ? 
```js
type === "password"
```

Reply:
done